backend/migrations/00074_add_ai_tutor_feature_flag.sql

```
@@ -0,0 +1,22 @@
+-- +goose Up
+-- +goose NO TRANSACTION
+ALTER TYPE feature ADD VALUE IF NOT EXISTS 'ai_tutor';
+
+INSERT INTO public.feature_flags (name, enabled)
+VALUES ('ai_tutor', FALSE)
+ON CONFLICT (name) DO NOTHING;
+
+-- +goose Down
+-- +goose NO TRANSACTION
+DELETE FROM public.facility_feature_flags WHERE feature = 'ai_tutor';
```

SQLFluff (4.3.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)